webapp/lib/controller.dart

```diff
@@ -317,3 +383,17 @@ void saveProjectConfiguration(Map config) {
   model.additionalProjectConfigurationLanguages.removeWhere((l) => languagesAdded.contains(l));
   loadProjectConfigurationView();
 }
+
+void savePackageConfiguration() {}
+
+// Helper Mehods
+
+Map _findActivePackageById(String packageId) {
+  return model.activePackages[packageId];
+}
+
+model.Configuration _findActivePackageConfigurationById(String packageId) {
+  return model.activePackages[packageId]['configurationData'];
+}
```

Comment:
These wrapper methods don't do very much anymore. I'd prefer we delete them and just access the model map directly.

Reply:
I agree. Removed